### ISSUE TITLE
Syntax error: Unterminated quoted string

### ISSUE THREAD

**Issue by priyadarshan** [cbsd/cbsd#100]
I just installed `cbsd` on a fresh FreeBSD 11-RELEASE, and followed the first tutorial here: https://www.bsdstore.ru/en/tutorial.html but it throws error `Syntax error: Unterminated quoted string`

```
root@kavi:~ # cbsd jconstruct-tui
[...]
/usr/jails/ftmp/jcreate_jconf.67746: 48: Syntax error: Unterminated quoted string
Config file for jconf: /usr/jails/ftmp/jail1.67455.jconf
```

`# cat /usr/jails/ftmp/jail1.67455.jconf`

```
# DO NOT EDIT THIS FILE. PLEASE USE INSTEAD:
# cbsd jconfig jname=jail1
relative_path="1";
jname="jail1";
path="/usr/jails/jails/jail1";
host_hostname="jail1.my.domain";
ip4_addr="DHCP";
mount_devfs="1";
allow_mount="1";
allow_devfs="1";
allow_nullfs="1";
mount_fstab="/usr/jails/jails-fstab/fstab.jail1";
arch="amd64";
mkhostsfile="1";
devfs_ruleset="4";
ver="11.0";
basename="";
baserw="0";
mount_src="0";
mount_obj="0";
mount_kernel="0";
mount_ports="1";
astart="1";
data="/usr/jails/jails-data/jail1-data";
vnet="0";
applytpl="1";
mdsize="0";
rcconf="/usr/jails/jails-rcconf/rc.conf_jail1";
floatresolv="1";
zfs_snapsrc="";

exec_poststart="0";
exec_poststop="";
exec_prestart="0";
exec_prestop="0";

exec_master_poststart="0";
exec_master_poststop="0";
exec_master_prestart="0";
exec_master_prestop="0";
pkg_bootstrap="1";
with_img_helpers="";
runasap="0";
interface="auto";
jailskeldir="/usr/jails/share/FreeBSD-jail-skel";
jail_profile="default";
# root password
user_pw_root=''';
emulator="jail"
```

**Comment by priyadarshan** [comment]
I have found issue: one cannot use password`'`

Admittedly, that is very unsafe, only good for quick testing.

Sorry for the noise!

Thank you for `bsd`, it is really excellent.

**Comment by olevole** [comment]
Got it. I'll think of something for extra check, Thanks for report!

**Comment by priyadarshan** [comment]
Thank you. It occurred to me that the issue will come anytime there is a quote sign inside the password, like `test'`, etc.
Let me know if I can be of any help testing.

**Comment by olevole** [comment]
It is corrected in 11.0.5: https://github.com/cbsd/cbsd/commit/2d94cebb760628cdedb8ffc0ecc0621a8e43b625
